src/main/webapp/location-form.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>location-form</title>
+</head>
+<body>
+
+<div class="padded rounded panel centered">
+    <h1 id="page-title">Location Form</h1>
+</div>
+
+<div class="padded rounded panel">
+    <form id="location-form" class="hidden" method="POST" enctype = "multipart/form-data">
```

Comment:
Does this form work without an action?

Also, you don't need multipart/form-data unless you're allowing the user to upload a file. For now, don't bother with that - we can do that next week.